modules/local/get_test_data.nf

```diff
@@ -4,15 +4,15 @@ process GET_TEST_DATA {
     container "docker.io/yuukiiwa/git:latest"
```

Comment:
Huh, interesting, this isn't how we normally retrieve test data with nf-core (either via URLs, or upstream step in the `ci.yml`), is there a reason why you do it like this?

Reply:
what's in the container?

Reply:
The container contains the `git` command. This is not required for most nf-core/test-dataset retrieval, but it is required here as I need to stage the entire fast5 directory containing many fast5 files to test the basecalling and RNA modification detection functionalities